definitions/checks/candlepin/validate_db.rb

```diff
@@ -12,7 +12,8 @@ class ValidateDb < ForemanMaintain::Check
     def run
       result, result_msg = feature(:candlepin_database).execute_cpdb_validate_cmd
       next_steps = []
-      if feature(:downstream) && feature(:downstream).current_minor_version == '6.2'
+      if feature(:downstream) && feature(:downstream).satellite? &&
```

Comment:
Why not `feature(:satellite) && feature(:satellite).current_minor_version == '6.2'`?